knowledge_repo/app/app.py

```diff
@@ -126,7 +126,7 @@ def update_typeahead_data(post):
 
             # For every tag in the excluded tags, create the tag object if it doesn't exist
             # To ensure that posts with the excluded tags do not show up in the typeahead
-            excluded_tags = current_app.config.get('EXCLUDED_TAGS')
+            excluded_tags = current_app.config.get('EXCLUDED_TAGS') or []
```

Comment:
sounds good, I was initially going to do that but noticed elsewhere in the code you had defaulted to this paradigm 👍